src/main/java/com/bunq/sdk/model/generated/DraftPayment.java

```diff
@@ -0,0 +1,270 @@
+
+    return processForId(responseRaw);
+  }
+
+  public static BunqResponse<Integer> update(ApiContext apiContext, Map<String, Object> requestMap,
+      Integer userId, Integer monetaryAccountId, Integer draftPaymentId) {
+    return update(apiContext, requestMap, userId, monetaryAccountId, draftPaymentId,
+        new HashMap<>());
+  }
+
+  /**
+   * Update a DraftPayment.
+   */
+  public static BunqResponse<Integer> update(ApiContext apiContext, Map<String, Object> requestMap,
+      Integer userId, Integer monetaryAccountId, Integer draftPaymentId,
+      Map<String, String> customHeaders) {
+    ApiClient apiClient = new ApiClient(apiContext);
+    byte[] requestBytes = gson.toJson(requestMap).getBytes();
+    BunqResponseRaw responseRaw = apiClient
+        .put(String.format(ENDPOINT_URL_UPDATE, userId, monetaryAccountId, draftPaymentId),
+            requestBytes, customHeaders);
+
+    return processForId(responseRaw);
+  }
+
+  public static BunqResponse<List<DraftPayment>> list(ApiContext apiContext, Integer userId,
+      Integer monetaryAccountId) {
+    return list(apiContext, userId, monetaryAccountId, new HashMap<>());
+  }
+
+  public static BunqResponse<List<DraftPayment>> list(ApiContext apiContext, Integer userId,
+      Integer monetaryAccountId, Map<String, String> params) {
+    return list(apiContext, userId, monetaryAccountId, params, new HashMap<>());
+  }
+
+  /**
+   * Get a listing of all DraftPayments from a given MonetaryAccount.
+   */
+  public static BunqResponse<List<DraftPayment>> list(ApiContext apiContext, Integer userId,
+      Integer monetaryAccountId, Map<String, String> params, Map<String, String> customHeaders) {
+    ApiClient apiClient = new ApiClient(apiContext);
+    BunqResponseRaw responseRaw = apiClient
+        .get(String.format(ENDPOINT_URL_LISTING, userId, monetaryAccountId), params, customHeaders);
+
+    return fromJsonList(DraftPayment.class, responseRaw, OBJECT_TYPE);
+  }
+
+  public static BunqResponse<DraftPayment> get(ApiContext apiContext, Integer userId,
+      Integer monetaryAccountId, Integer draftPaymentId) {
+    return get(apiContext, userId, monetaryAccountId, draftPaymentId, new HashMap<>());
+  }
+
+  /**
+   * Get a specific DraftPayment.
+   */
+  public static BunqResponse<DraftPayment> get(ApiContext apiContext, Integer userId,
+      Integer monetaryAccountId, Integer draftPaymentId, Map<String, String> customHeaders) {
+    ApiClient apiClient = new ApiClient(apiContext);
+    BunqResponseRaw responseRaw = apiClient
+        .get(String.format(ENDPOINT_URL_READ, userId, monetaryAccountId, draftPaymentId),
+            new HashMap<>(), customHeaders);
+
+    return fromJson(DraftPayment.class, responseRaw, OBJECT_TYPE);
+  }
+
+  /**
+   * The id of the DraftPayment.
+   */
+  public Integer getId() {
+    return this.id;
+  }
+
+  public void setId(Integer id) {
+    this.id = id;
+  }
+
+  /**
+   * The id of the MonetaryAccount the DraftPayment applies to.
```

Comment:
Please choice upper or lower case  for Id

Reply:
generated